From: Karten Böddeker <freifunk@kb-light.de>
Date: Thu, 15 Oct 2015 16:35:06 +0200
Subject: [PATCH] add support for futro

diff --git a/patches/openwrt/1000-Add-support-for-futro.patch b/patches/openwrt/1000-Add-support-for-futro.patch
new file mode 100644
index 0000000..9efa901
--- /dev/null
+++ b/patches/openwrt/1000-Add-support-for-futro.patch
@@ -0,0 +1,13 @@
+From: Karsten Böddeker <freifunk@kb-light.de>
+Date: Thu, 15 Oct 2015 14:40:53 +0200
+Subject: Add support for futro
+
+diff --git a/target/linux/x86/generic/config-default b/target/linux/x86/generic/config-default
+index 2babaab..12d1c7f 100644
+--- a/target/linux/x86/generic/config-default
++++ b/target/linux/x86/generic/config-default
+@@ -198,3 +198,4 @@
+ # CONFIG_X86_LONGHAUL is not set
+ # CONFIG_X86_PCC_CPUFREQ is not set
+ CONFIG_X86_PM_TIMER=y
++CONFIG_PATA_ATIIXP=y
diff --git a/targets/x86-generic/profiles.mk b/targets/x86-generic/profiles.mk
index b6551a7..a8a068d 100644
--- a/targets/x86-generic/profiles.mk
+++ b/targets/x86-generic/profiles.mk
@@ -1,4 +1,6 @@
-X86_GENERIC_NETWORK_MODULES := kmod-3c59x kmod-e100 kmod-e1000 kmod-e1000e kmod-natsemi kmod-ne2k-pci kmod-pcnet32 kmod-8139too kmod-r8169 kmod-sis900 kmod-tg3 kmod-via-rhine kmod-via-velocity kmod-forcedeth
+X86_GENERIC_NETWORK_MODULES := kmod-3c59x kmod-e100 kmod-e1000 kmod-e1000e kmod-natsemi kmod-ne2k-pci kmod-pcnet32 kmod-8139too kmod-r8169 kmod-sis900 kmod-tg3 kmod-via-rhine kmod-via-velocity kmod-forcedeth \
+       kmod-usb-core kmod-usb2 kmod-usb-hid kmod-usb-net kmod-usb-net-asix kmod-usb-net-rtl8150 kmod-usb-net-rtl8152 kmod-usb-net-dm9601-ether
+


 $(eval $(call GluonProfile,GENERIC,$(X86_GENERIC_NETWORK_MODULES)))
--
2.1.4
